thinc/backends/ops.py

```diff
@@ -1254,6 +1254,9 @@ def position_encode(
         numpy_ops = NumpyOps()
         return self.asarray2f(numpy_ops.position_encode(N, D, period, out))
 
+    def gather_add(self, table: Floats2d, indices: Ints2d) -> FloatsXd:
```

Comment:
Isn't the return value `Floats2d`?
(if it is - we don't need the `cast` in `hashembed.py`)

Reply:
Yeah, fixed!